dockerworker/dockerimage_test.go

```diff
@@ -0,0 +1,62 @@
+// +build docker
+
+package dockerworker
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	imageTaskID   = "Xx0aPfyOTU2o_0FZnr_AJg"
```

Comment:
What is this taskID, can you provide a comment?

Reply:
I added a comment explaining what it is.

Reply:
We already have such tasks, and their task definitions are stored in /testdata/tasks - please update that directory and add a note here, if this is needed.

Reply:
This is not the same as the `taskID` in the main tests, this is a known task containing the various artifact images formats for tests.